src/blockchain/Blockchain.ts

```diff
@@ -47,56 +52,108 @@ export interface IBlockchainApi {
     identity: Identity,
     tx: SubmittableExtrinsic
   ): Promise<SubmittableExtrinsic>
+  submitSignedTx(
+    identity: Identity,
+    tx: SubmittableExtrinsic,
+    opts?: SubscriptionPromiseOptions
+  ): Promise<SubmittableResult>
   submitTx(
     identity: Identity,
     tx: SubmittableExtrinsic,
     opts?: SubscriptionPromiseOptions
   ): Promise<SubmittableResult>
-  getNonce(accountAddress: string): Promise<Codec>
+  getNonce(accountAddress: string): Promise<BN>
+  reSignTx(
+    identity: Identity,
+    tx: SubmittableExtrinsic
+  ): Promise<SubmittableExtrinsic>
 }
+const TxOutdated = '1010: Invalid Transaction: Transaction is outdated'
+const TxPriority = '1014: Priority is too low:'
+const TxAlreadyImported = 'Transaction Already'
 
+export const IS_RELEVANT_ERROR: ErrorEvaluator = (err: Error) => {
+  return new RegExp(
+    `${TxAlreadyImported}|${TxOutdated}|${TxPriority}`,
+    'g'
+  ).test(err.message)
+}
 export const IS_READY: ResultEvaluator = (result) => result.status.isReady
 export const IS_IN_BLOCK: ResultEvaluator = (result) => result.isInBlock
 export const EXTRINSIC_EXECUTED: ResultEvaluator = (result) =>
   ErrorHandler.extrinsicSuccessful(result)
 export const IS_FINALIZED: ResultEvaluator = (result) => result.isFinalized
+export const IS_USURPED: ResultEvaluator = (result) =>
+  result.status.isUsurped && ERROR_TRANSACTION_USURPED()
+export const IS_ERROR: ResultEvaluator = (result) => {
+  return (
+    (result.status.isDropped && Error('isDropped')) ||
+    (result.status.isInvalid && Error('isInvalid')) ||
+    (result.status.isFinalityTimeout && Error('isFinalityTimeout'))
+  )
+}
+export const EXTRINSIC_FAILED: ResultEvaluator = (result) => {
+  return (
+    ErrorHandler.extrinsicFailed(result) &&
+    (ErrorHandler.getExtrinsicError(result) || UNKNOWN_EXTRINSIC_ERROR)
+  )
+}
 
-export const IS_ERROR: ResultEvaluator = (result) =>
-  result.isError && ERROR_UNKNOWN()
-export const EXTRINSIC_FAILED: ResultEvaluator = (result) =>
-  ErrorHandler.extrinsicFailed(result) &&
-  (ErrorHandler.getExtrinsicError(result) || UNKNOWN_EXTRINSIC_ERROR)
```

Comment:
oh, did this not work right? weeeeird

Reply:
I didn't remove this, it's working just the way you wrote it.